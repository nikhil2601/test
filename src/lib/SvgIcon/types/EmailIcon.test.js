import React from 'react';
import { shallow } from 'enzyme';

import EmailIcon from './EmailIcon';

describe('<EmailIcon />', () => {
    const wrapper = shallow(<EmailIcon />);

    it('should render correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
