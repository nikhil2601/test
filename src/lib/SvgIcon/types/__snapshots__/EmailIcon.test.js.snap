// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmailIcon /> should render correctly 1`] = `
<SvgIcon
  color="inherit"
  height="1rem"
  margin={null}
  opacity={null}
  padding={null}
  titleAccess={null}
  viewBox="0 0 24 24"
  width="1rem"
>
  <path
    d="M20,8L12,13L4,8V6L12,11L20,6M20,4H4C2.89,4 2,4.89 2,6V18A2,2 0 0,0 4,20H20A2,2 0 0,0 22,18V6C22,4.89 21.1,4 20,4Z"
  />
</SvgIcon>
`;
